package.json

```diff
@@ -8,7 +8,9 @@
   },
   "scripts": {
     "test": "NODE_ENV=test `npm bin`/mocha --require @babel/register --require spec/javascripts/spec_helper.js 'spec/javascripts/**/**_spec.js'",
-    "build": "true"
+    "build": "true",
+    "snyk-protect": "snyk protect",
+    "prepare": "yarn run snyk-protect"
```

Comment:
Ok so adding in this in last time resulted in some breakage in our build process.... I am not sure we're ready to add it yet....